node/pkg/watchers/cosmwasm/watcher.go

```diff
@@ -126,6 +129,8 @@ func NewWatcher(
 		latestBlockURL = "cosmos/base/tendermint/v1beta1/blocks/latest"
 	}
 
+	b64Encoded := chainID != vaa.ChainIDInjective && (chainID != vaa.ChainIDTerra2 || env == common.UnsafeDevNet)
```

Comment:
can we do `env == common.UnsafeDevNet || (chainID != vaa.ChainIDInjective && chainID != vaa.ChainIDTerra2)`

Reply:
Changed